org.eclipse.m2e.site/pom.xml

```diff
@@ -36,6 +36,61 @@
   </build>
 
   <profiles>
+	<profile>
```

Comment:
Please comment here about why this is necessary (ie "Eclipse SimRel requires all artifacts to be jar-signed. So we sign the BND artifacts and update the p2 metadata accordingly").